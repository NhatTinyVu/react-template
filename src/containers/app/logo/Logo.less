.logo {
  height: 64px;
  float: left;

  img {
    height: 100%;
  }
}
