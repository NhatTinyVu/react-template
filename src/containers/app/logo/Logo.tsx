import React from 'react';
import { Link } from 'react-router-dom';
import styles from './Logo.less';

const Logo: React.FC = () => (
  <Link to="/">
    <div className={styles.logo}>
      <img src={`${process.env.PUBLIC_URL}/logo.svg`} alt="logo" />
    </div>
  </Link>
);

export default Logo;
