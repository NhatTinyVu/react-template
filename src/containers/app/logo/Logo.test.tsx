import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Logo from '.';

it('renders correctly', (): void => {
  const wrapper = shallow(<Logo />);
  expect(toJSON(wrapper)).toMatchSnapshot();
});
