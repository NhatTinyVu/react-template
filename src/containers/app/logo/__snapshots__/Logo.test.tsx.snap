// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="logo"
>
  <img
    alt="logo"
    src="/logo.svg"
  />
</div>
`;
